dhall-yaml/dhall-yaml.cabal

```
@@ -37,8 +37,8 @@ Library
         base                      >= 4.8.0.0  && < 5   ,
         aeson                     >= 1.0.0.0  && < 1.5 ,
         bytestring                               < 0.11,
-        dhall                     >= 1.27.0   && < 1.28,
-        dhall-json                >= 1.5.0    && < 1.6 ,
+        dhall                     >= 1.28.0   && < 1.29,
+        dhall-json                >= 1.5.0    && < 1.7 ,
```

`dhall-yaml` relies on some changes that I implemented as a part of the package split.

```suggestion
        dhall-json                >= 1.6.0    && < 1.7 ,
```